tests/test_basic_data.py

```diff
@@ -16,6 +17,7 @@
 ## Class DataBunch
 
 def test_DataBunch_Create():
+    this_tests(DataBunch.create)
```

Comment:
I remind again that we need to use live objects, so all these should be `object.foo`, not `class.foo` - `data.foo` in this case. and all of the remaining changes in this file.

Also please use a separate PR, otherwise this PR will never finish.